modules/node_modules/@ncigdc/utils/formatFileSize.js

```diff
@@ -1,31 +1,4 @@
 // @flow
+import filesize from 'filesize';
```

Comment:
It happened to serve as an abstraction layer similar to the `<XXLinks>`s and our `withRouter`

I'm uncertain about how concrete the specs would remain for filesize formatting. We used to use base 10 (explicitly opting for it in an instance) to calculate filesize but showed `KB` (signaling base 2) instead of `kB` (signaling base 10) as the unit; there was one place that was directly using filesize instead of this utility function, and thus showed the correct unit format of `kB` instead of `KB`. The ticket asked that the `kB` be changed to `KB`, potentially unaware that those are 2 different units  

I asked in slack but no one responded so I made an out-of-spec assumption that it's not a big deal to change from using base 10 to base 2 to satisfy the ticket's requirement of having uppercase units, which also allows our desired result to happily align with the default options of the library; but if specs change (either it's important to use base 10, or perhaps even important to use base 10 and have uppercase units), it'd be convenient to have a single place where modifications need to be made

It's not a DAL I would have created starting out from scratch, but the cost/benefit seemed to work out in favour of keeping the module and avoiding bloating the PR's diff. 

Don't feel strongly about it though, don't mind replacing all instances if we're fairly certain this format will remain acceptable :)